frontend/app/routes/$lang+/_public+/apply+/$id+/applicant-information.tsx

```diff
@@ -50,14 +55,25 @@ export async function loader({ request, params }: LoaderFunctionArgs) {
 export async function action({ request, params }: ActionFunctionArgs) {
   const applyFlow = getApplyFlow();
   const { id } = await applyFlow.loadState({ request, params });
+  const t = await getFixedT(request, handle.i18nNamespaces);
+
+  /**
+   * Schema applicant information.
+   */
+  const applicantInformationSchema: z.ZodType<ApplicantInformationState> = z.object({
```

Comment:
Use `z.string().trim();` with strings to remove whitespaces before calling `.min(1)`.

Reply:
I've noticed that we can only return the message string as second argument instead of `{ message: "Error here!" }` object.